tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -1732,13 +1736,13 @@ class PlaceholderExpansionScanner {
             OuterExpr = E;
           }
         }
-        return true;
+        return Action::Continue(E);
       }
 
-      bool walkToExprPost(Expr *E) override {
+      PostWalkResult<Expr *> walkToExprPost(Expr *E) override {
         if (E->getStartLoc() == TargetLoc)
-          return false; // found what we needed to find, stop walking.
-        return true;
+          return Action::Stop(); // found what we needed to find, stop walking.
+        return Action::Continue(E);
```

Comment:
Could use `Action::StopIf` here if you want

Reply:
Oh, thanks for the tip! but let me merge this as is for now 🙏